### ISSUE TITLE
Warn on missing alt-text

### ISSUE THREAD

**Issue by hadley** [r-lib/pkgdown#2357]
I'm not 100% this is the right place, but it would be useful if pkgdown would warn when images in your vignettes lack alt text.

(It would be nice to do this for examples too, but I'm not currently aware of a way to set alt text from examples)
